dev/dvm/bin/dvm

```diff
@@ -11,6 +11,6 @@ require "dvm"
 
 begin
   DVM::Application.start(ARGV)
-rescue DVM::DVMArgumentError => e
```

Comment:
Thanks @marcparadise @sdelano now that the underlying raise is working in `dvm/lib/dvm.rb` this isn't needed, so I put it back.